eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/model/DbsnpVariantAllelesTest.java

```diff
@@ -189,14 +194,29 @@ public void noCountSTRAlleles() {
                                                                      DbsnpVariantType.MICROSATELLITE);
 
         assertEquals("AT", emptyCountOnly.getReferenceInForwardStrand());
-        assertEquals(Arrays.asList(""), emptyCountOnly.getAllelesInForwardStrand());
+        assertEquals(Arrays.asList("AT"), emptyCountOnly.getAllelesInForwardStrand());
 
-        DbsnpVariantAlleles dashAndOtherCounts = new DbsnpVariantAlleles("AC", "(A)(AC)2/(A)1(AC)", Orientation.FORWARD,
-                                                                         Orientation.FORWARD,
-                                                                         DbsnpVariantType.MICROSATELLITE);
+        DbsnpVariantAlleles emptyAndOtherCounts = new DbsnpVariantAlleles("AC", "(A)(AC)2/(A)1(AC)",
+                                                                          Orientation.FORWARD, Orientation.FORWARD,
+                                                                          DbsnpVariantType.MICROSATELLITE);
 
-        assertEquals("AC", dashAndOtherCounts.getReferenceInForwardStrand());
-        assertEquals(Arrays.asList("ACAC", "A"), dashAndOtherCounts.getAllelesInForwardStrand());
+        assertEquals("AC", emptyAndOtherCounts.getReferenceInForwardStrand());
+        assertEquals(Arrays.asList("AACAC", "AAC"), emptyAndOtherCounts.getAllelesInForwardStrand());
+    }
```

Comment:
For some reason, test fails for https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs208705307.

Reply:
Also tests fail for https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs109798407